ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -87,26 +96,25 @@ service_ip=$(echo "${octets[*]}" | sed 's/ /./g')
 # Determine appropriete subject alt names
 sans="IP:${cert_ip},IP:${service_ip},DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.${dns_domain},DNS:${master_name}"
 
-curl -L -O https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz > /dev/null 2>&1
-tar xzf easy-rsa.tar.gz > /dev/null
```

Comment:
Right you are, sir.  It did actually still work because modern GNU tar at least will autodetect compression formats.